go/op_exporter/k8sClient/main.go

```diff
@@ -0,0 +1,20 @@
+package k8sClient
+
+import (
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+func Newk8sClient() (client *kubernetes.Clientset, err error) {
+	// creates the in-cluster config
+	config, err := rest.InClusterConfig()
+	if err != nil {
+		panic(err.Error())
+	}
+	// creates the clientset
+	client, err = kubernetes.NewForConfig(config)
+	if err != nil {
+		panic(err.Error())
+	}
+	return client, err
```

Comment:
It feels a bit more idiomatic to explicitly return `client, nil` here instead of `err` which is known to be `nil` at this point after reading the code

Reply:
Agreed, updated!

Reply:
Are you sure that you pushed the commit that does this?